src/webgl/material.js

```diff
@@ -127,11 +130,25 @@ p5.prototype.loadShader = function(
  * // the 'varying's are shared between both vertex & fragment shaders
  * let varying = 'precision highp float; varying vec2 vPos;';
  *
+ * Shaders can alter the positioning of shapes drawn with them.
+ * To ensure consistency in rendering, it's recommended to use the vertex shader, as in the example below.
+ *
  * // the vertex shader is called for each vertex
- * let vs =
- *   varying +
- *   'attribute vec3 aPosition;' +
- *   'void main() { vPos = (gl_Position = vec4(aPosition,1.0)).xy; }';
+ *let vs = `
+ * uniform mat4 uModelViewMatrix;
+ * uniform mat4 uProjectionMatrix;
+ *
+ * attribute vec3 aPosition;
+ * attribute vec2 aTexCoord;
+ * varying vec2 vTexCoord;
+ *
+ * void main() {
+ *   vTexCoord = aTexCoord;
+ *   vec4 positionVec4 = vec4(aPosition, 1.0);
+ *   gl_Position = uProjectionMatrix * uModelViewMatrix * positionVec4;
+ *  }
+ *`;
```

Comment:
To add consistent indentation:
```suggestion
 * `;
```